grafana/dashboards.libsonnet

```diff
@@ -25,11 +25,11 @@
 
   // add a new dashboard, creating the folder if necessary
   addDashboard(name, dashboard, folder=''):: {
-    local hasShards = std.objectHas(super.folders[name], 'shards'),
+    local hasShards = std.objectHas(super.grafanaDashboardFolders, folder) && std.objectHas(super.grafanaDashboardFolders[folder], 'shards'),
```

Comment:
I'm not sure if this is a concern but an empty string is a valid key:
```
$ cat main.jsonnet 
local obj = {
  '': 'test',
};
$ jsonnet main.jsonnet 
{
  has: std.objectHas(obj, ''),
  value: obj[''],
  obj: obj,
}
```

Reply:
Yep, thats fine  this is just preventing breakage.